cheevos/cheevos.c

```diff
@@ -2866,17 +2866,18 @@ static int cheevos_iterate(coro_t* coro)
       /* Negative values because CORO_SUB generates positive values */
       SNES_MD5    = -1,
       GENESIS_MD5 = -2,
-      NES_MD5     = -3,
-      GENERIC_MD5 = -4,
-      EVAL_MD5    = -5,
-      FILL_MD5    = -6,
-      GET_GAMEID  = -7,
-      GET_CHEEVOS = -8,
-      LOGIN       = -9,
-      HTTP_GET    = -10,
-      DEACTIVATE  = -11,
-      PLAYING     = -12,
-      DELAY       = -13,
+      LYNX_MD5    = -3,
+      NES_MD5     = -4,
+      GENERIC_MD5 = -5,
+      EVAL_MD5    = -6,
+      FILL_MD5    = -7,
+      GET_GAMEID  = -8,
+      GET_CHEEVOS = -9,
+      LOGIN       = -10,
+      HTTP_GET    = -11,
+      DEACTIVATE  = -12,
+      PLAYING     = -13,
+      DELAY       = -14,
```

Comment:
Small nit - drop the comma at the very end of the list. This will otherwise cause problems for C89.